README.rst

```diff
@@ -159,3 +156,15 @@ the options table must be regenerated using the Makefile:
 
     $ make update-docs
 
+CLI options
+***********
+
+The following options can be supplied when running CLI commands:
+
+=======  ==============  ===================================================================================================================================
+Key      Default         Description
+=======  ==============  ===================================================================================================================================
+address  localhost:1323  Address of the remote node. Must contain at least host and port, URL scheme may be omitted. In that case it 'http://' is prepended.
+timeout  10s             Client time-out when performing remote operations.
```

Comment:
my guess it uses unix style time strings, like 10s, 1m, etc? Would be good to add to description